scripts/update_version.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+# Copyright 2019 Uber Technologies, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This script is used interactively as part of the H3 release process 
+# (RELEASE.md) to update the version number in the VERSION file. Before
+# writing the new version to the file, changelog information is presented
+# for verification.
+
+set -eo pipefail
+
+CURRENT_VERSION=$(<VERSION)
```

Comment:
For my info, this requires that we run the script from the root, right? It won't work if we run from inside `scripts`?

That might suggest it would be worthwhile to have a Make target for this.

Reply:
Yes, this expects to be run from the root of the repo, where the `VERSION` file is located.

Normally I'd run `make` from outside the repo directory, so I could add a parameter for the source directory that should be updated.

Reply:
Updated to include this.